sshnp_docker_demo/db-template.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#
+# DO NOT RUN THIS SCRIPT. THIS IS MEANT TO BE RUN BY ANOTHER SCRIPT IN ONE OF THE THREE DIRECTORIES
+#
+
+if [ -z "$1" ]
+then
+    echo "Usage: ./db.sh <sshnp/sshnpd/sshrvd>"
+    exit 1
+fi
+
+rm Dockerfile
+cp ../Dockerfile .
+docker container prune -f
```

Comment:
This is pretty rude. People might have images and containers lying around that would be destroyed here.

Reply:
Removed prunes https://github.com/atsign-foundation/at_demos/pull/97/commits/769faa5f523903b90824d6e07539b11de2b2fd62

Reply:
Removed prunes https://github.com/atsign-foundation/at_demos/pull/97/commits/769faa5f523903b90824d6e07539b11de2b2fd62